ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -12,6 +12,7 @@ export interface BurnAssetRequest {
   account: string
   assetId: string
   fee: string
+  feeRate?: string
```

Comment:
I think it makes sense to make `fee` optional here -- if a client passes a fee rate they shouldn't need to also pass a fee

Reply:
donie